Druid/DRCogitOperandSorter.class.st

```diff
@@ -198,6 +198,11 @@ DRCogitOperandSorter >> visitJump: aDRJump [
 	"Nothing"
 ]
 
+{ #category : #visiting }
+DRCogitOperandSorter >> visitLeftRotate: aDRRotate [ 
+	^ self
+]
```

Comment:
Looking at the LeftShift implementation:

```st
DRCogitOperandSorter >> visitLeftShift: aDRLeftShift [ 

	"Shift is not commutative!
	=> extract its first operand into a copy instruction"
	self extractFirstOperandInCopyIfNecessaryFrom: aDRLeftShift
]
```

I think that we should do the same here.